boto3-assume/aws/GdsEc2Client.py

```diff
@@ -0,0 +1,19 @@
+# GdsEc2Client
+# extends GdsAwsClient 
+# implements aws ec2 api queries 
+from aws.GdsAwsClient import GdsAwsClient
```

Comment:
Python file names (and so module names) should be lower case and separated by underscore.
Example: `GdsAwsClient` --> `gds_aws_client`